client/src/components/AuthNavButton/AuthNavButton.tsx

```diff
@@ -0,0 +1,24 @@
+import Button from '@material-ui/core/Button';
+import Box from '@material-ui/core/Box';
+import { Link } from 'react-router-dom';
+import useStyles from './useStyles';
+
+interface Props {
+  linkTo: string;
+  btnText: string;
+  variant: any;
+}
+
+const AuthNavButton = ({ linkTo, btnText, variant }: Props): JSX.Element => {
```

Comment:
👍🏽 

Also, we can safely delete the AuthHeader component as it is identical to this component.